client/src/common/constants/enums.js

```diff
@@ -65,3 +65,15 @@ export const PasswordValidationValues = Object.freeze({
   MIN: 4,
   MAX: 32
 });
+
+export const EmailNotificationsFrequency = Object.freeze({
```

Comment:
why not create by enumerable?

Reply:
enumerable doesn't work here 'cycle error' from eslint is displayed.